lib/ReactViews/StandardUserInterface/map-column.scss

```diff
@@ -18,7 +18,6 @@
 
 .map__cell {
   display: table-cell;
-  position: relative;
```

Comment:
This was the key line that made the map reappear.  Hopefully removing it doesn't break IE or something crazy.  It seemed ok in my testing.  @AlexGilleran